completion/available/gradle.completion.bash

```diff
@@ -66,10 +67,15 @@ __gradle-generate-script-cache() {
 	local cache_ttl_mins=${GRADLE_CACHE_TTL_MINUTES:-30240}
 	local script_exclude_pattern=${GRADLE_COMPLETION_EXCLUDE_PATTERN:-"/(build|integTest|out)/"}
 
-	if [[ ! $(find $cache_dir/$cache_name -mmin -$cache_ttl_mins 2> /dev/null) ]]; then
+	if [[ ! $(find "$cache_dir/$cache_name" -mmin "-$cache_ttl_mins" 2> /dev/null) ]]; then
 		# Cache all Gradle scripts
-		local gradle_build_scripts=$(find $project_root_dir -type f -name "*.gradle" -o -name "*.gradle.kts" 2> /dev/null | grep -E -v "$script_exclude_pattern")
-		printf "%s\n" "${gradle_build_scripts[@]}" > $cache_dir/$cache_name
+		local gradle_build_scripts
+		gradle_build_scripts=$(
+			find "$project_root_dir" -type f \
+				\( -name "*.gradle" -or -name "*.gradle.kts" \) 2> /dev/null \
```

Comment:
`-or` is not defined by POSIX, which means that some `find` implementations doesn't support it.

```suggestion
				\( -name "*.gradle" -o -name "*.gradle.kts" \) 2> /dev/null \
```